pywbemtools/pywbemcli/_cmd_instance.py

```diff
@@ -833,15 +835,18 @@ def cmd_instance_modify(context, instancename, options):
 
     If successful, this operation returns nothing.
     """
+    context.spinner_stop()
```

Comment:
Is there a reason for this spinner_stop() call or is this a remainder of a debug session?

Reply:
Arrgh hangover from debug session.  Actually should be applied for every case of the if verbose pattern to stop spinner before outputting the message.

Reply:
Note that this did point out an issue where I was not consisently turning off spinner before those verbose outputs.  **Fixed**